controls/docker_level2.rb

```diff
@@ -163,7 +163,7 @@
   ref 'https://docs.docker.com/engine/reference/run/#security-configuration'
   ref 'http://wiki.apparmor.net/index.php/Main_Page'
 
-  only_if { ['ubuntu', 'debian'].include? os[:family] }
+  only_if { %w(ubuntu debian).include? os[:family] }
```

Comment:
`os[:family]` will be identical for ubuntu and debian. We should use `os.name` instead